src/services/transfer/__tests__/transfer-out-ehr-core.test.js

```diff
@@ -52,60 +51,58 @@ describe('transferOutEhrCore', () => {
     payload: 'payload XML with updated referenced fragment message ids'
   };
 
-  patientAndPracticeOdsCodesMatch.mockResolvedValue(true);
 
-  updateMessageIdForEhrCore.mockImplementation(ehrCore => ehrCoreWithUpdatedMessageId);
+  afterEach(() => {
+    jest.resetAllMocks();
+  })

```

Comment:
Super small optional one, can we slap a semicolon on the end of this please? :D

```suggestion
  afterEach(() => {
    jest.resetAllMocks();
  });
```